.travis.yml

```diff
@@ -33,11 +33,11 @@ matrix:
       env: GROUP=examples EXTRA_CMD=--no-js
 
     # linting, package and docs building (python 3.4)
-    - python: 3.4
+    - python: 3.6
       env: GROUP=flake_docs
 
     # integration tests (python 3.4)
```

Comment:
Most of `.travis.yml` is going away in a follow on PR so not going to bother fixing at this point.